packages/d3fc-webgl/src/shaders/vertexShaderSnippets.js

```diff
@@ -24,3 +24,14 @@ export const point = {
            gl_PointSize = vSize + uEdgeSize + 1.0;
            gl_Position = vec4(aXValue, aYValue, 0, 1);`
 };
+
+export const square = {
+    header: `attribute float aXValue;
+        attribute float aYValue;
+        attribute float aSize;
+        uniform float uEdgeSize;
```

Comment:
I appreciate you're just copying the pattern here but I think we call `edgeSize` `strokeWidth` in SVG/canvas. Is it the same thing and if so can we switch over?

Reply:
I think it is the same thing, I'll change it to `strokeWidth`